web/src/osm2streets-svelte/layers/DynamicConnectedRoads.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  import type { GeoJSON } from "geojson";
+  import Layer from "../Layer.svelte";
+  import LayerControls from "../LayerControls.svelte";
+  import { hoveredIntersection, network } from "../store";
+
+  let gj: GeoJSON | undefined = undefined;
+  let show = true;
+  $: if ($network && $hoveredIntersection) {
```

Comment:
All of the layers are reactive, changing usually when the loaded StreetNetwork does. And in this case, based on hover state